cmd/capmcd/bmcapi.go

```diff
@@ -344,11 +365,35 @@ func (d *CapmcD) doBmcPowerCall(call bmcCall) bmcPowerRc {
 			return res
 		}
 	}
+
+	var sessionAuthToken string
+	if len(sessionAuthPath) > 0 {
+		req, err := http.NewRequest("POST", sessionAuthPath, bytes.NewBuffer([]byte(sessionAuthBody)))
+		req.SetBasicAuth(ni.BmcUser, ni.BmcPass)
+		req.Header.Set("Accept", "*/*")
+		req.Header.Set("Content-Type", "application/json")
+		// execute the request
+		rfClientLock.RLock()
+		rsp, err := d.rfClient.Do(req)
+		rfClientLock.RUnlock()
+		if err != nil {
+			log.Printf("POST %s\n Body           --> %s\n %s Network Error: %s",
+				sessionAuthPath, sessionAuthBody, ni.BmcType, err)
```

Comment:
I don't believe the sessionAuthBody has been sanitized at this point, logging it will expose the credentials.

Reply:
Removed body